### DISCUSSION TITLE
API only says 404

### DISCUSSION THREAD

**Discussion by Jak2k** [mastodon/mastodon#22730] [Q&A] [answered]
I'm currently playing with the mastodon-api.

The web-ui makes a call to the api:
`https://mastodontech.de/api/v1/statuses/109573649635351471/favourite`

When I make the call manually  with the browser, I get a 404. I would understand when I got a 401. But a 404 ?!

All other things (only see the status or getting timelines) work as expected.

**Comment by nightpool** [comment, marked as answer]
when you send a GET instead of a POST, rails won't be able to find a matching route, which leads to a 404 error. This is because Rails can't distinguish between a path that exists with a non-supported method and a method that exists with a non-existent path.

**Reply by Jak2k** [reply]
Thanks!
